pyml tries to lookup the correct libpython for the binary by
parsing ldd output but doesn't handle OpenBSD's ldd

Index: bundles/pyml/pyml-current/py.ml
--- bundles/pyml/pyml-current/py.ml.orig
+++ bundles/pyml/pyml-current/py.ml
@@ -191,7 +191,7 @@ let uninit_pythonpath () =
       end
 
 let ldd executable =
-  let command = Printf.sprintf "ldd %s" executable in
+  let command = Printf.sprintf "ldd %s | awk '/libpython/ {print $7}'" executable in
   match run_command_opt command false with
     None -> []
   | Some lines ->
